sympy/core/equation.py

```diff
@@ -0,0 +1,359 @@
+"""
+Algebraic Equations with SymPy
+==============================
+
+author: Jonathan Gutow <gutow@uwosh.edu>
+
+Overrides of binary binary operations based on suggestions from [Oscar Benjamin](https://github.com/oscarbenjamin) 
+
+date: May 2020
+
+license: GPL V3+
```

Comment:
I don't think we want any GPL in sympy because the rest of sympy is not GPL.

SymPy's license file already has "Copyright (c) 2006-2020 SymPy Development Team" and then the BSD license.

Reply:
Sorry, I will pull that out. It is left over from my personal repository for the bare python file. I missed that and some extra white spaces and other formatting stuff. I will fix and update the pull request.

Reply:
Please also remove the author and data metadata. The authors are listed in the AUTHORS file, which is generated automatically. Information about who authored which specific pieces of code or when they were authored can be ascertained from the git history.

Reply:
I believe I have pulled all the offending lines from the doc strings.